src/praisonai-train/praisonai_train/cli/commands/benchmark.py

```
@@ -0,0 +1,101 @@
+        targets, n=n, concurrency=concurrency, base=base or None,
+        recipe=cfg.get("recipe", "tamil"), prompt=cfg.get("prompt"),
+        max_tokens=cfg.get("max_tokens", DEFAULT_MAX_TOKENS),
+        api_version=cfg.get("api_version", DEFAULT_API_VERSION),
+        request_timeout=cfg.get("request_timeout", 180),
+        on_result=_live,
+    )
+
+    from praisonai_train.cli.output.console import get_output_controller
+    get_output_controller().print_table(
+        _TABLE_HEADERS, [_row_cells(r) for r in results],
+        title=f"speed benchmark (ranked by rows/min)",
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)